### ISSUE TITLE
Part Config File Import Issue

### ISSUE THREAD

**Issue by 13Stewartc** [BeamNG/Blender-JBeam-Editor#26]
I tried to import the t83_custom (and multiple other us_semi configs) and none of them seem to load anything besides a single object named us_semi

![image](https://github.com/user-attachments/assets/0e1b6682-96fe-4366-99da-db29326ff31e)

**Comment by angelo234** [comment]
Ah yeah because the JBeam parser isn't compatible with the new JBeam language (e.g. slots2, components)

**Comment by 13Stewartc** [comment]
Ah, that explains it, will support be added for this at some point ? @angelo234 I coulda used it yesterday haha :D Things ended up being far more complex then they needed to be as a result of this 😅

**Comment by angelo234** [comment]
Yeah I don't have much time nowadays since I'm working on other things. I'm not sure when I'll have time to work on this. :/

**Comment by 13Stewartc** [comment]
Ah, okay. Provided its done at some point i cant see it being that much of an issue. :) 

But presumably from now on any new vehicles (or ground up remasters) will most likely be using the slots2 / components (new jbeam additions) so whilst its not that big a deal now, it will be in time. But no rush really, I understand being busy :D Have a good rest of your day mate 👍

**Comment by angelo234** [comment]
Thanks you too :)
